web/modules/custom/talkcodemodule/src/Plugin/Block/talkcodemoduleblock.php

```diff
@@ -0,0 +1,83 @@
+<?php
+
+namespace Drupal\talkcodemodule\Plugin\Block;
+
+use Drupal\Core\Block\BlockBase;
+use Drupal\Core\Form\FormStateInterface;
+
+/**
+ * Provides a 'talkcodemoduleblock' block.
+ *
+ * @Block(
+ *  id = "talkcodemoduleblock",
+ *  admin_label = @Translation("Talkcodemoduleblock"),
+ * )
+ */
+class talkcodemoduleblock extends BlockBase {
+
+
+  /**
+   * {@inheritdoc}
+   */
+  public function defaultConfiguration() {
+    return [
+         'title' => $this->t('BUILD IT BOLD AND BEAUTIFUL'),
+         'subtitle' => $this->t('Work with a reliable developer and build a brilliant digital experience'),
+         'talkcodebutton' => $this->t('LETS TALK CODE'),
```

Comment:
@brandtziegler  There are different ways to handle this, but what I typically do is not make the text all caps and instead handle that in CSS. If you consider the text being in all caps part of the styling, then it technically should be handled in the CSS.

I'm not requesting that you change anything here, just sharing my thoughts.